tools/cli/go-whisk/whisk/api.go

```diff
@@ -257,3 +338,245 @@ func (s *ApiService) Delete(api *Api, options *ApiOptions) (*http.Response, erro
     return nil, nil
 }
 
+/////////////
+// V2 Cmds //
+/////////////
+func (s *ApiService) ListV2(apiListOptions *ApiListRequestOptions) (*ApiListResponseV2, *http.Response, error) {
+    route := "web/whisk.system/apimgmt/getApi.http"
+
+    routeUrl, err := addRouteOptions(route, apiListOptions)
+    if err != nil {
+        Debug(DbgError, "addRouteOptions(%s, %#v) error: '%s'\n", route, apiListOptions, err)
+        errMsg := wski18n.T("Unable to add route options '{{.options}}'",
+            map[string]interface{}{"options": apiListOptions})
+        whiskErr := MakeWskErrorFromWskError(errors.New(errMsg), err, EXITCODE_ERR_GENERAL, DISPLAY_MSG,
+            NO_DISPLAY_USAGE)
+        return nil, nil, whiskErr
+    }
+    Debug(DbgInfo, "Api GET/list route with api options: %s\n", routeUrl)
+
+    req, err := s.client.NewRequestUrl("GET", routeUrl, nil, DoNotIncludeNamespaceInUrl, AppendOpenWhiskPathPrefix, EncodeBodyAsJson)
+    if err != nil {
+        Debug(DbgError, "http.NewRequestUrl(GET, %s, nil, DoNotIncludeNamespaceInUrl, AppendOpenWhiskPathPrefix, EncodeBodyAsJson) error: '%s'\n", routeUrl, err)
```

Comment:
I think a separate issue is needed to address the consolidation of duplicate error message generation.  This PR is following the current pattern.